src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -39,6 +39,10 @@ public function validate($password, Constraint $constraint)
             throw new UnexpectedTypeException($constraint, __NAMESPACE__.'\UserPassword');
         }
 
+        if (null === $password || '' === $password) {
+            return;
```

Comment:
- PHP 5.6.30
- FOSUB 1.3.7
- validation error is gone after updating from Symfony 2.7.29 to 2.7.30